src/test/datascience/testHelpers.tsx

```diff
@@ -59,29 +59,41 @@ async function testInnerLoop(
 
 export function runDoubleTest(
     name: string,
-    testFunc: (type: 'native' | 'interactive') => Promise<void>,
+    testFunc: (this: Mocha.Context, type: 'native' | 'interactive') => Promise<void>,
```

Comment:
Fixes to properly pass the context/scope, so we can use `this.skip`.
We had `c.skip` but that didn't work as the scope wasn't setup properly.